src/Zforce.cpp

```diff
@@ -20,10 +20,12 @@
 
 #include <string.h>
 #include <inttypes.h>
-#include "I2C/I2C.h"
 #include "Zforce.h"
-#if USE_I2C_LIB == 0
-  #include <Wire.h>
+#if USE_I2C_LIB == 1
+  #include "I2C/I2C.h" // for "official" SAMD21-based "Neonode Prototyping Board" AND Arduino boards ( gotta test but I didn't try yet with 128 bytes buff in I2C.h just modded )
```

Comment:
This comment is incorrect. The I2C.h library does not work with SAMD21-based arduinos, i.e Adafruit Feather M0 or the Neonode prototyping board. The SAMD21 boards compleley rely on the Wire.h library as those boards do not have the 32 byte buffer limit as the ATMega328 and similar MCUs.